admin/includes/languages/english/template_select.php

```diff
@@ -1,44 +1,32 @@
 <?php
-//
-// +----------------------------------------------------------------------+
-// |zen-cart Open Source E-commerce                                       |
-// +----------------------------------------------------------------------+
-// | Copyright (c) 2003 The zen-cart developers                           |
-// |                                                                      |
-// | http://www.zen-cart.com/index.php                                    |
-// |                                                                      |
-// | Portions Copyright (c) 2003 osCommerce                               |
-// +----------------------------------------------------------------------+
-// | This source file is subject to version 2.0 of the GPL license,       |
-// | that is bundled with this package in the file LICENSE, and is        |
-// | available through the world-wide-web at the following url:           |
-// | http://www.zen-cart.com/license/2_0.txt.                             |
-// | If you did not receive a copy of the zen-cart license and are unable |
-// | to obtain it through the world-wide-web, please send a note to       |
-// | license@zen-cart.com so we can mail you a copy immediately.          |
-// +----------------------------------------------------------------------+
-//  $Id: template_select.php 1105 2005-04-04 22:05:35Z birdbrain $
-//
+/**
+ * @package admin
+ * @copyright Copyright 2003-2020 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id:  Modified in v1.5.7 $
+ */
 
 define('HEADING_TITLE', 'Template Selection');
-
+define('TEXT_TEMPLATE_SELECT_INFO', '<p>Here you may change the template used to format the Shopfront.</p><p>A different template may be assigned to each of the installed languages. The default template is used when there is no specific template assigned to a language.</p><p>It is possible to view a template in a <em>private</em> session, for example when testing a new template: see the online Help <strong>?</strong>.</p>');

```

Comment:
A few thoughts:

- While I accept that terminologies are different in various parts of the world, the rest of Zen Cart uses the word "store" instead of "shop".
For consistency can we change "Shopfront" to "storefront"?

- the 2nd sentence, about "a different template per language" and "the default is used...": Can we put the "default" part first, and reword the rest so that it sounds like one will never need to do the language-association of templates?
I don't want to encourage people to start making new template-language associations in case we change that in 1.5.8.